hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/AbstractFindTargetGreedy.java

```diff
@@ -55,10 +56,12 @@ public abstract class AbstractFindTargetGreedy implements FindTargetStrategy {
   protected AbstractFindTargetGreedy(
       ContainerManager containerManager,
       PlacementPolicy placementPolicy,
+      PlacementPolicy ecPlacementPolicy,
       NodeManager nodeManager) {
     sizeEnteringNode = new HashMap<>();
     this.containerManager = containerManager;
     this.placementPolicy = placementPolicy;
+    this.ecPlacementPolicy = ecPlacementPolicy;
```

Comment:
I think it might be better to not have different placement policy fields? We need a better way to fetch these placement policies.